site/src/pages/TemplatesPage/TemplatesPageView.tsx

```diff
@@ -121,113 +123,122 @@ export const TemplatesPageView: FC<React.PropsWithChildren<TemplatesPageViewProp
-                          : Language.emptyViewNoPerms
-                      }
-                      descriptionClassName={styles.emptyDescription}
-                      cta={<CodeExample code="coder templates init" />}
-                    />
-                  </TableCell>
+                  <TableCell width="50%">{Language.nameLabel}</TableCell>
+                  <TableCell width="16%">{Language.usedByLabel}</TableCell>
+                  <TableCell width="16%">{Language.lastUpdatedLabel}</TableCell>
+                  <TableCell width="16%">{Language.createdByLabel}</TableCell>
+                  <TableCell width="1%"></TableCell>
                 </TableRow>
-              ) : (
-                props.templates?.map((template) => {
-                  const templatePageLink = `/templates/${template.name}`
-                  const hasIcon = template.icon && template.icon !== ""
+              </TableHead>
+              <TableBody>
+                <Maybe condition={Boolean(props.loading)}>
+                  <TableLoader />
+                </Maybe>
 
-                  return (
-                    <TableRow
-                      key={template.id}
-                      hover
-                      data-testid={`template-${template.id}`}
-                      tabIndex={0}
-                      onKeyDown={(event) => {
-                        if (event.key === "Enter") {
-                          navigate(templatePageLink)
-                        }
-                      }}
-                      className={styles.clickableTableRow}
-                    >
-                      <TableCellLink to={templatePageLink}>
-                        <AvatarData
-                          title={template.name}
-                          subtitle={template.description}
-                          highlightTitle
-                          avatar={
-                            hasIcon && (
-                              <div className={styles.templateIconWrapper}>
-                                <img alt="" src={template.icon} />
-                              </div>
-                            )
+                <ChooseOne>
+                  <Cond condition={empty}>
+                    <TableRow>
+                      <TableCell colSpan={999}>
+                        <EmptyState
+                          message={Language.emptyMessage}
+                          description={
+                            props.canCreateTemplate
+                              ? Language.emptyDescription
+                              : Language.emptyViewNoPerms
                           }
+                          descriptionClassName={styles.emptyDescription}
+                          cta={<CodeExample code="coder templates init" />}
                         />
-                      </TableCellLink>
+                      </TableCell>
+                    </TableRow>
+                  </Cond>
+                  <Cond condition>
```

Comment:
Why condition here does not have a value?

Reply:
The last `Cond` is the fallback, like in a switch statement - if the other conditions are not met, this will render regardless of whether its condition is true. So I thought the best convention would be to write it this way so that the reader knows it's not a real condition. The alternative is to make `condition` optional and omit it here, but if it were accidentally omitted in an earlier `Cond`, that one would always be skipped, and I thought avoiding that bug was important.